lib/registries/addon/components/registries-navbar/component.ts

```diff
@@ -1,27 +1,49 @@
 import { tagName } from '@ember-decorators/component';
-import { action } from '@ember/object';
+import { action, computed } from '@ember/object';
 import { and } from '@ember/object/computed';
 import RouterService from '@ember/routing/router-service';
 import { inject as service } from '@ember/service';
-import Media from 'ember-responsive';
+import { camelize } from '@ember/string';
+import Features from 'ember-feature-flags/services/features';
+import config from 'ember-get-config';
 
 import { layout } from 'ember-osf-web/decorators/component';
+import RegistrationProviderModel from 'ember-osf-web/models/registration-provider';
 import defaultTo from 'ember-osf-web/utils/default-to';
+import Media from 'ember-responsive';
 import { AuthBase } from 'osf-components/components/osf-navbar/auth-dropdown/component';
 import { OSF_SERVICES } from 'osf-components/components/osf-navbar/component';
-import config from 'registries/config/environment';
+import registriesConfig from 'registries/config/environment';
 
 import template from './template';
 
-const { externalLinks } = config;
+const { externalLinks } = registriesConfig;
+const {
+    featureFlagNames: {
+        egapAdmins,
+    },
+} = config;
 
 @tagName('')
 @layout(template)
 export default class RegistriesNavbar extends AuthBase {
     @service media!: Media;
     @service router!: RouterService;
+    @service features!: Features;
+
+    provider?: RegistrationProviderModel;
 
     @and('media.isMobile', 'searchDropdownOpen') showSearchDropdown!: boolean;
+    @computed('provider.{acceptingSubmissions,id}')
```

Comment:
should match what's used:
```suggestion
    @computed('provider.{allowSubmisions,id}')
```